app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
Yeah this is full-spec.
You can have only ``onDestroy()`` and ``onCreate()``. 👍 

``onPause`` and ``onResume`` use case:

* Enabling GPS only foreground
* Some model's tasks suspending in background

``onSaveInstanceState `` and ``onRestoreInstanceState `` use case:

* (As you know) save or load states when screen rotating